sandbox/src/mocks/cancelReferral/responses/ExampleCancelledReferralWithAppointment.json

```diff
@@ -24,6 +24,12 @@
                             "reference": "#appointment"
                         }
                     },
+                    {
+                        "url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-Appointment-Reference-1",
+                        "valueReference": {
+                          "reference": "Appointment/70009"
```

Comment:
Should this be "Appointment/70001", given the appointment id below is 70001?

Reply:
{
            "name": "updatedAppointment",
            "resource": {
                "id": "70001",
                "meta": {
                    "versionId": "1",
                    "profile": [

Reply:
Yup it should be, good spot 🙂